apps/studio/pages/project/[ref]/storage/vectors/buckets/[bucketId].tsx

```diff
@@ -25,7 +23,6 @@ const VectorsBucketPage: NextPageWithLayout = () => {
     <div className="storage-container flex flex-grow p-4">
       {isError && <StorageBucketsError error={error as any} />}
 
-      {isLoading && <GenericSkeletonLoader className="w-full" />}
```

Comment:
As mentioned elsewhere: I'm following a pattern from https://github.com/supabase/supabase/pull/39610 where we put the loading at the section level rather than blocking the page. That's the idea, anyway!